tests/rsa/mod.rs

```diff
@@ -201,6 +209,7 @@ fn rsa_jwk() {
 // https://jwt.io/ is often used for examples so ensure their example works with jsonwebtoken
 #[cfg(feature = "use_pem")]
 #[test]
+#[wasm_bindgen_test]
```

Comment:
Do we really need to put that on every test? :(

Reply:
I think this is unavoidable. Couldn't find aything about the reusability of the standard `#[test]` macro.